iroh-js/src/gossip.rs

```diff
@@ -75,21 +76,19 @@ impl From<SubscribeResponse> for Message {
 /// Iroh gossip client.
 #[napi]
 pub struct Gossip {
-    node: Iroh,
+    gossip: Arc<iroh_gossip::net::Gossip>,
 }
 
 #[napi]
 impl Iroh {
     /// Access to gossip specific funtionaliy.
     #[napi(getter)]
     pub fn gossip(&self) -> Gossip {
-        Gossip { node: self.clone() }
-    }
-}
-
-impl Gossip {
-    fn client(&self) -> &iroh::client::Iroh {
-        self.node.inner_client()
+        let gossip = self
+            .router
+            .get_protocol(iroh_gossip::net::GOSSIP_ALPN)
+            .expect("no gossip available");
```

Comment:
Just calling out that we are removing `get_protocol` in the next release.

Reply:
...i guess this actually doesn't matter since we are removing the protocols in ffi next release anyway.